### ISSUE TITLE
pretrained models for batch normalized version of vgg

### ISSUE THREAD

**Issue by bodokaiser** [pytorch/vision#152]
According to [here](https://github.com/alexgkendall/caffe-segnet/issues/48) we should be able to reuse the retrained weight from e.g. `vgg16` and apply them to `vgg16bn` we will just need to write a small conversion (as order in `features` differs due to additional `nn.BatchNorm2d` layers).

Do you agree? Are PR welcome on this?

**Comment by colesbury** [comment]
I don't think you can take the weights from the model without batch norm and stick them in the model with batch normalization. The activation statistics will almost certainly be wrong at each batch-normalized layer if someone tries fine-tuning the model or otherwise continue training.

**Comment by bodokaiser** [comment]
Could we then also offer pretained weights for the batch normalized versions?

**Comment by wangg12** [comment]
+1 for the request.
